css/moonstone-variables.less

```diff
@@ -194,7 +194,7 @@
 @moon-button-small-text-size: 24px;
 @moon-button-large-min-width: 180px;
 @moon-button-small-min-width: 132px;
-@moon-button-h-padding: 18px;
+@moon-button-h-padding: 30px;
```

Comment:
Lets change this to {{27px}} so the small round button (one character) stays round.